x/meshsecurity/keeper/querier_test.go

```diff
@@ -39,15 +39,52 @@ func TestQueryVirtualStakingMaxCap(t *testing.T) {
 	}
 	for name, spec := range specs {
 		t.Run(name, func(t *testing.T) {
-			gotRsp, gotErr := NewQuerier(keepers.EncodingConfig.Marshaler, k).VirtualStakingMaxCap(sdk.WrapSDKContext(ctx), &types.QueryVirtualStakingMaxCapRequest{
+			gotRsp, gotErr := NewQuerier(keepers.EncodingConfig.Marshaler, k).VirtualStakingMaxCapLimit(sdk.WrapSDKContext(ctx), &types.QueryVirtualStakingMaxCapLimitRequest{
 				Address: spec.addr,
 			})
 			if spec.expErr {
 				require.Error(t, gotErr)
 				return
 			}
 			require.NoError(t, gotErr)
-			assert.Equal(t, spec.expAmount, gotRsp.Limit)
+			assert.Equal(t, spec.expAmount, gotRsp.Cap)
 		})
 	}
 }
+
+func TestQueryVirtualStakingMaxCapLimits(t *testing.T) {
+	// setup
+	ctx, keepers := CreateDefaultTestInput(t)
+	k := keepers.MeshKeeper
+
+	querier := NewQuerier(keepers.EncodingConfig.Marshaler, k)
+	// when
+	gotRsp, err := querier.VirtualStakingMaxCapLimits(sdk.WrapSDKContext(ctx), &types.QueryVirtualStakingMaxCapLimitsRequest{})
+	// then
+	require.NoError(t, err)
+	assert.Zero(t, len(gotRsp.MaxCapInfos))
```

Comment:
fyi: The error message is more verbose with `assert.Len(t, gotRsp.MaxCapInfos), 0)`